rllib/core/learner/learner.py

```diff
@@ -886,7 +883,7 @@ def compute_loss_for_module(
         self,
         *,
         module_id: ModuleID,
-        config: Optional["AlgorithmConfig"] = None,
+        config: "AlgorithmConfig",
```

Comment:
Do we now always need to provide a config? I think for most algorithms this is not needed because `self.config` should be available.

Reply:
I felt like this is the better solution for users. 2 reasons:
* Users normally override `compute_loss_for_module`, so now they do NOT have to implement a logic, where config is None.
* Users do NOT normally override the more top-level `compute_loss`, so we can easily provide each module's individual config through our base implementations.

In other words, if we had left this arg to be optional, every user writing a custom loss function would have had to implement a (not too known) logic on how to get the module's individual config.